pkg/app/web/src/components/settings-page/piped/index.tsx

```diff
@@ -91,6 +94,11 @@ export const SettingsPipedPage: FC = memo(function SettingsPipedPage() {
     selectFilteredPipeds(state, filterValues.enabled)
   );
 
+  const [isUpgradeDialogOpen, setUpgradeDialogOpen] = useState(false);
+  const handleUpgradeDialogClose = useCallback(() => {
+    setUpgradeDialogOpen(false);
+  }, []);
```

Comment:
```suggestion
  const handleUpgradeDialogClose = () => setUpgradeDialogOpen(false);
```
since there are no dependencies here, no need to use `useCallback`.

Reply:
Nice catch!